extensions/amp-sidebar/1.0/toolbar.js

```diff
@@ -32,6 +35,12 @@ export class Toolbar {
     /** @private {!Object} **/
     this.win_ = win;
 
+    /** @private {Element|null} */
```

Comment:
If you drop the null here you get:

found   : (Element|null)
required: Element
          setStyles(this.body_, {